WordPress/Classes/ViewRelated/Site Creation/Web Address/AddressTableViewCell.swift

```diff
@@ -160,10 +177,25 @@ final class AddressTableViewCell: UITableViewCell {
             let string = index == 0 ? tag.localizedString : "\n\(tag.localizedString)"
             attributedString.append(.init(string: string, attributes: attributes))
         }
+
+        switch cost {
+        case .freeWithPaidPlan:
+            let firstYearAttributes: [NSAttributedString.Key: Any] = [
+                .foregroundColor: Appearance.saleCostTextColor,
+                .font: Appearance.smallCostFont
```

Comment:
Yes, you're right. We're using `NSAttributedString` with `UIFont` attributes. It requires the view to be reopened for font size changes to react or to listen to notifications and rebuild attributed string. However, some labels do not have a specific font set for attributed text, and have font only set on `UILabel`. This way the dynamic type changes work immediately. 

I made an update to rely on the font set directly on the label rather than on the attributed text where we can. I want to avoid listening to notifications in the `UITableViewCell`. Therefore, in the most common cases we now react to dynamic type changes immediately:

https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/8f7a2697-30b0-4ba6-8559-03551985dd90